### PR TITLE
Allow application to run on mainnet or testnet

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: daemon/src/maker.rs [hunk 1]
```diff
@@ -72,6 +68,48 @@ struct Opts {
     /// If enabled logs will be in json format
     #[clap(short, long)]
     json: bool,
+
+    #[clap(subcommand)]
+    network: Network,
```
Comment (bonomat) [inline daemon/src/maker.rs:73]: If there is no default, you might need to adapt the CI and want to update: `start_all.sh`

**Review by bonomat — APPROVED** [review]
Nice, we are getting closer 🥳

File: daemon/src/maker.rs (hunk 1 above)
Comment (thomaseizinger) [reply daemon/src/maker.rs:73]: Good catch!

**Pull request merged**